contrib/ffmpeg/module.defs

```diff
@@ -194,6 +194,7 @@ endif
 
 ifeq (1,$(FEATURE.mf))
 FFMPEG.CONFIGURE.extra += \
+      --enable-hwaccels \
```

Comment:
Maybe enable only the hwaccel that will be used, instead of all of them.

Reply:
Fixed.